btree/btree.c

```diff
@@ -2690,32 +2688,30 @@ static void lock_op_unlock(struct td *tree)
 	m0_rwlock_write_unlock(&tree->t_lock);
 }
 
-static struct m0_btree_oimpl *level_alloc(int height)
+static void level_alloc(struct m0_btree_oimpl *oi, int height)
```

Comment:
Even if we pass struct level it won't help, because then we anyway have to return struct level*,
Here I am passing oi as parameter to avoid returning level pointer, If we do not want to pass struct m0_btree_oimpl *oi,, then we have to return level pointer from function.